roles/kubernetes/kubeadm/tasks/main.yml

```diff
@@ -53,11 +53,10 @@
     timeout: 60
 
 - name: Update server field in kubelet kubeconfig
-  replace:
-    path: "{{ kube_config_dir }}/kubelet.conf"
-    regexp: '(\s+)https://{{ first_kube_master }}:{{ kube_apiserver_port }}(\s+.*)?$'
-    replace: '\1{{ kube_apiserver_endpoint }}\2'
-    backup: yes
+  lineinfile:
+    dest: "{{ kube_config_dir }}/kubelet.conf"
+    regexp: 'server:'
+    line: '    server: {{ kube_apiserver_endpoint }}'
```

Comment:
should be localhost:8080 ? or what's the variable for the local nginx ?

Reply:
if loadbalancer_apiserver_localhost is true.  The kube_apiserver_endpoint‘s value will point to the local nginx address